### ISSUE TITLE
Building v2.0.6 from sources on Ubuntu 20.04

### ISSUE THREAD

**Issue by adibacco** [RedisJSON/RedisJSON#655]
```
git clone https://github.com/RedisJSON/RedisJSON.git
cd RedisJSON
git checkout v2.0.6
# Modify Cargo.toml adding    cargo-features = ["edition2021"] on top of file
 cargo build --release

```

and then I get an error:

```
   Compiling redisjson v2.0.6 (/home/antodib/packages/RedisJSON)
error: the option `Z` is only accepted on the nightly compiler

error: could not compile `redisjson`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by gkorland** [comment]
@adibacco which rust version are you using?

**Comment by adibacco** [comment]
I’m using rustc version 1.53.0

**Comment by gkorland** [comment]
Can you please try to upgrade to rust 1.58.1?

> I tried to download the .so module (v2) for Ubuntu from
> https://redis.com/redis-enterprise-software/download-center/modules/
> but it is not available for Ubuntu 20.04, this seems strange.

The Ubuntu 18.04 should work on 20.04
